concordium-sdk/src/main/java/com/concordium/sdk/transactions/TransactionType.java

```diff
@@ -27,14 +27,7 @@ public enum TransactionType {
     ENCRYPTED_TRANSFER_WITH_MEMO((byte) 23),
     TRANSFER_WITH_SCHEDULE_AND_MEMO((byte) 24),
     CONFIGURE_BAKER((byte) 25),
-    CONFIGURE_DELEGATION((byte) 26),
-
-    /**
-     * This exists only to deserialize Raw Account Transaction Payload.
-     * Will be removed in future versions.
-     * todo: Deserialize payload using raw byte array.
-     */
-    RAW_PAYLOAD((byte) 255);
```

Comment:
No this is no more required as the `TransactionType transactionType` property from the `AccountTransaction` (`AbstractAccountTransaction`) has been removed. 
Reason being the Transaction Type should be encoded with `byte[] payloadBytes`. 

Earlier `AccountTransaction` derived from `AbstractAccountTransaction` which had `TransactionType transactionType` property. Forcing Account Transaction to populate it. But since Account Transaction has no defined type RAW was used instead.